com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/TestModule.java

```diff
@@ -51,9 +55,13 @@ protected void configure() {
     bind(LanguageServer.class).to(TestLanguageServer.class);
     bind(LanguageEngineFacade.class).to(CobolLanguageEngineFacade.class);
     bind(WorkspaceService.class).to(MockWorkspaceService.class);
+
+    bind(FileSystemService.class).to(FileSystemServiceImpl.class);
+
     bind(Communications.class).to(ServerCommunications.class);
     bind(TextDocumentService.class).to(MyTextDocumentService.class);
     bind(CobolTextRegistry.class).to(ZipTextRegistry.class);
+    bind(AbstractDataBusBroker.class).to(DefaultDataBusBroker.class);
```

Comment:
By idea, this binding should be from the widest to the most possibly precise, i.e. DataBusBroker -> DefaultDataBusBroker